src/gui/systray.h

```diff
@@ -34,6 +34,7 @@ class Systray : public QSystemTrayIcon
 {
     Q_OBJECT
 public:
+    Systray(QObject *parent = nullptr);
```

Comment:
Btw, one can do `using QSystemTrayIcon::QSystemTrayIcon;`  to inherit the constructor.